azkaban-common/src/test/java/azkaban/project/JdbcProjectImplTest.java

```diff
@@ -352,6 +397,7 @@ public void clearDB() {
       dbOperator.update("DELETE FROM project_flows");
       dbOperator.update("DELETE FROM project_files");
       dbOperator.update("DELETE FROM project_events");
+      dbOperator.update("DELETE FROM project_flow_files");
```

Comment:
nit: truncate should be faster than delete

Reply:
Thanks for sharing the knowledge. I think truncate is better in this case.